exp/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
Should this be a symlink instead? Or is there any way to just have any referenced uses use the existing boilerplate?

Reply:
It could but it shouldn't change either, I can change it if you prefer

Reply:
I'm not overly worried about it if you're not. I always cringe a bit when I see replicated content.